### ISSUE TITLE
Consider lazy loading intercom

### ISSUE THREAD

**Issue by aholachek** [5calls/5calls#575]
It currently accounts for half the site's JS weight and more than half of the CPU time used to render the site: 
<img width="556" alt="Screen Shot 2020-03-22 at 9 41 13 PM" src="https://user-images.githubusercontent.com/3680488/77271804-fc71fb80-6c85-11ea-9ea8-5a7b960a6655.png">

**Comment by aholachek** [comment]
<img width="1583" alt="Screen Shot 2020-03-22 at 9 43 48 PM" src="https://user-images.githubusercontent.com/3680488/77271898-3f33d380-6c86-11ea-87c7-e7bd46fa4e5b.png">

**Comment by nickoneill** [comment]
Agreed! We're using this unofficial react plugin for intercom which is not very customizable, but I will see if we can make it async in our branch here: https://github.com/5calls/react-intercom
